src/src/single-spa-angular.ts

```diff
@@ -15,7 +15,7 @@ const defaultOpts = {
   updateFunction: () => Promise.resolve(),
 };
 
-export default function singleSpaAngular(userOpts: SingleSpaAngularOpts): LifeCycles {
+export function singleSpaAngular(userOpts: SingleSpaAngularOpts): LifeCycles {
```

Comment:
:warning: breaking change for `@4` because angular's package format and ng-packagr don't allow default exports.

Reply:
:'( that sucks